sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -716,18 +647,26 @@ export class EventHubSender extends LinkEntity {
             this.name
           );
 
-          this._sender!.on(SenderEvents.accepted, onAccepted);
-          this._sender!.on(SenderEvents.rejected, onRejected);
-          this._sender!.on(SenderEvents.modified, onModified);
-          this._sender!.on(SenderEvents.released, onReleased);
-
-          const delivery = this._sender!.send(message, undefined, 0x80013700);
-          log.sender(
-            "[%s] Sender '%s', sent message with delivery id: %d",
-            this._context.connectionId,
-            this.name,
-            delivery.id
-          );
+          try {
+            const delivery = await this._sender!.send(message, undefined, 0x80013700);
```

Comment:
I'm curious, why don't we resolve with the `delivery` object? This seems like it could be useful for a customer to have access to.

@ramya-rao-a

Reply:
Agree, not sure if there was a specific reason/advantage behind for having API this way

Reply:
It is an internal representation of the message which is very specific to what rhea does. We didnt find much use for it, so we don't resolve it.